### ISSUE TITLE
Tracking physical location message

### ISSUE THREAD

**Issue by Uafinder** [Kademi/kademi-dev#1412]
![image](https://cloud.githubusercontent.com/assets/1945230/13505918/1718022c-e184-11e5-9caa-b3a07faeb2fa.png)
Is it ok? Or this is just a:

**Comment by bradmac** [comment]
Yes, the quick lead feature records your location. This is for use with smart phones, for when sales people meet a potential customer, eg at a conference, and they take a photo of their business card or make an audio recording. The location is recorded to help them remember the details later when they transcribe the lead.
